### PR TITLE
Feature/add helper

### PR DESCRIPTION
**Pull request by BelehovEgor**

### CODE REVIEW

**Review by MariyaSin — APPROVED** [review]

File: src/Kernel/Helpers/GetEnvironmentVariableHelper.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using LT.DigitalOffice.Kernel.Exceptions.Models;
+using System;
+
+namespace LT.DigitalOffice.Kernel.Helpers
+{
+    public static class GetEnvironmentVariableHelper
+    {
+        public static string Get(string key)
```
Comment (djino97) [inline src/Kernel/Helpers/GetEnvironmentVariableHelper.cs:8]: rename Get => GetVariable

File: src/Kernel/Helpers/GetEnvironmentVariableHelper.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        {
+            return Environment.GetEnvironmentVariable(key)
+                ?? throw new NotFoundException($"No variable named {key}");
```
Comment (djino97) [inline src/Kernel/Helpers/GetEnvironmentVariableHelper.cs]: not sure that exception right

**Review by djino97 — APPROVED** [review]

**Review by TIove — APPROVED** [review]

File: LT.DigitalOffice.Kernel.nuspec [hunk 3]
```diff
@@ -2,34 +2,35 @@
...
+    <version>1.1.28</version>
     <title>LT.DigitalOffice.Kernel</title>
     <authors>LT Digital Office</authors>
     <owners>LT Digital Office</owners>
     <projectUrl>https://github.com/LT-Students/DigitalOffice-Kernel</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>Library used in the Digital Office project.</description>
     <releaseNotes>
-1.1.27 - Reworked AutoInjectRequest attribute
+      1.1.28 - Added GetEnvironmentVariableHelper and updated ConstStrings
```
Comment (TIove) [inline LT.DigitalOffice.Kernel.nuspec]: Strange spaces

File: src/Kernel/Helpers/GetEnvironmentVariableHelper.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/Kernel/Helpers/GetEnvironmentVariableHelper.cs:8]: name of class GetEnvironmentVariableHelper. I think it's clear that we waiting for in the Get method.

File: src/Kernel/Helpers/GetEnvironmentVariableHelper.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/Kernel/Helpers/GetEnvironmentVariableHelper.cs]: What do you want? What type of exception?

**Pull request merged**